docs/pages/machine-id/access-guides/databases.mdx

```diff
@@ -140,114 +104,129 @@ supported `format` options is below:
 | Generic TLS   | `tls`         | Provides `tls.key`, `tls.crt`, and `tls.cas` for generic clients that require specific file extensions. |
 
-Be sure to configure the `token` and `ca_pins` fields to match the output from
-`tctl bots add ...`.
+Outputs must be configured with a destination. In this example, the `directory`
+destination will be used. This will write artifacts to a specified directory on
+disk. Ensure that this directory can be written to by the Linux user that
+`tbot` runs as, and that it can be read by the Linux user that will be accessing
+applications.
 
-Machine ID also allows you to use Linux ACLs to control access to certificates
-on disk. You will use this to ensure only your application has access to the
-short-lived certificates Machine ID uses.
+Modify your `tbot` configuration to add a `database` output:
 
-We'll work with the assumption you will be running Machine ID as the Linux user
-`teleport` and your application as the Linux user `app`. Create and initialize
-the output destination by running this `tbot init` command either as `root`
-or as the `teleport` user:
-
-```code
-$ tbot init \
-    -c /etc/tbot.yaml \
-    --init-dir=/opt/machine-id \
-    --bot-user=teleport \
-    --owner=teleport:teleport \
-    --reader-user=app
+```yaml
+outputs:
+- type: database
+  destination:
+    type: directory
+    path: /opt/machine-id
+  # Specify the details of the database you wish to connect to.
+  service: example-server
+  database: example
+  username: alice
+  # Specify a format to use for the output credentials. For most databases,
+  # this configuration field can be omitted. See the table below for the
+  # databases that require a value to be specified here.
+  # format: mongo
 ```
 
-(!docs/pages/includes/machine-id/machine-id-init-bot-data.mdx!)
+If operating `tbot` as a background service, restart it. If running `tbot` in
+one-shot mode, it must be executed before you attempt to execute the Ansible
+playbook.
 
-<Notice type="note">
-Be sure to re-run `tbot init ...` as shown here if config templates are added
-or removed from `tbot.yaml`. You may run into permissions errors if `tbot init`
-is not run for new files.
-</Notice>
+## Step 3/4. Configure the database access local proxy
 
-Next, you will use systemd to run Machine ID in the background on your
-application node. Create a systemd.unit file at
-`/etc/systemd/system/machine-id.service`:
+Now that `tbot` has produced the database access credentials, a local proxy
+should be set up to forward the database connections from your database client
+to the Teleport Proxy through a TLS connection. This is necessary as the TLS
+connection allows the Teleport Proxy to identify the protocol and intended
+recipient.
 
-```systemd
-[Unit]
-Description=Teleport Machine ID Service
-After=network.target
+In addition, this local proxy needs to be running as long as the client needs to
+make connections to the database or as long as a connection is still open.
 
```

Comment:
I would move the example unit file and the paragraph that introduces it here. Otherwise, the preceding five paragraphs make it seem like the local proxy is an implementation detail left to the reader, rather than something we provide. 

I would also incorporate some of the architectural information about the local proxy into the instructions for using it.

Reply:
[5a3317b](https://github.com/gravitational/teleport/pull/32743/commits/5a3317b472d76a9e94fb2fc0594a33806b91cf78)